buoyant_force_from_volume_and_density.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+"""buoyant_force_from_volume_and_density.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1xlFQ4QkRMsWn8-7weSHc5qnnDE7sMwWQ
+"""
+
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input, validate_output)
+
+# Description
+## Any object immersed in liquid is affected by a buoyant force equal to the weight of the liquid displaced by it.
```

Comment:
Unfortunately this law is already implemented in symplyphysics: https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/dynamics/buoyant_force_from_density_and_volume.py